examples/mp-kv-sharding/python/solution.py

```diff
@@ -0,0 +1,99 @@
+from dslabmp import Context, Message, Process
+from typing import List
+
+
+class StorageProcess(Process):
```

Comment:
В данном случае процесс представляет узел хранилища, поэтому удобнее оставить как было

```suggestion
class StorageNode(Process):
```